modules/ROOT/pages/audit-logging.adoc

```diff
@@ -2099,13 +2099,30 @@ You can access the
 https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/[Audit Log REST API] from the Audit Logging Query API and its
 https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/organizations/8bfc8bbf-5508-419e-aadc-77dfe18a8172/assets/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/1.0.5/files/fat-raml/zip[RAML].
 
+
 === Use a Query Loop to make Requests in Audit Logs
 
-To gather data, you can initiate query loops and use cursors to navigate paginated audit logs. To avoid performance degradation, you can stop the query loop using the `total` by using the value returned by `IncludeTotal`.
+The https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/organizations/8bfc8bbf-5508-419e-aadc-77dfe18a8172/assets/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/1.0.5/files/fat-raml/zip[latest version of the query API] uses cursor pagination for efficiency, but you can continue to use the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/minor/1.0/console/method/%23245/[previous version of the query API], which uses offset pagination and returns the total number of records.
+
+
+The newer version of the API uses pagination with cursors, so you can initiate query loops using cursors. Setting the `cursorPagination` query parameter to true enables you to use cursor paging.
+When you use cursor paging, each response returns a cursor. The cursor corresponds to the last data entry in the response data set. The subsequent request in the query loop should use the cursor value that was returned in the previous response, and so on.
+When the data set is empty in response, you have reached the end of the query loop, and there is no more data to query for that time window. At that point, stop the query loop. When the data set is empty in the response, no cursor is returned.
+When you use cursor pagination, `total` is not returned by default, because the query loop is not dependent on the offset or the number of total rows.
+If you want to see the total when you use cursor paging, set the `doIncludeTotal` query parameter to true.
+
+When the `cursorPagination` query parameter is true and you use`offset` and `cursor` in the query body, `cursor` is prioritized, and `offset` is ignored.
+
+
+There is no mapping between `offset` and `cursor` values; And so when you run a query loop over a time window, you cannot switch to the new param somewhere in the middle by using the `cursorPagination` and continue the loop. This is because when using `cursorPagination`, `offset` is ignored and so the loop will be start from the beginning.
 
-The following example shows a series of calls in a query loop:
+This cursor pagination api is not applicable to the csv query, and that continues to use the offset based pagination.
```

Comment:
@sarathecoubian 
I added this line 'This cursor pagination api is not applicable to the csv query, and that continues to use the offset based pagination.'
The csv based api doesn't use the cursor based pagination and it'll continue to use the old offset based paging api.

Reply:
```suggestion
The cursor-based pagination API is not applicable to the CSV query. The query continues to use offset-based pagination.
```